### ISSUE TITLE
Publish a new version on the Terraform registry 

### ISSUE THREAD

**Issue by orf** [xunleii/terraform-module-k3s#79]
Hey!
This is a very handy module, thank you!

Would it be possible to update the version on the terraform registry with the latest changes? Specifically the `use_sudo` argument (added in #57) would be handy to have.

For anyone else seeing this, you can pull the latest master with:

```hcl
module "k3s" {
  source      = "github.com/xunleii/terraform-module-k3s.git?ref=45e073c6ea4784ded7260467a9f6f0f6e6f8f636"
 
  ...
}
```

**Comment by xunleii** [comment]
Hi @orf, happy new years !

Yep, sorry to be late, I will try to release a new version with #80 this week.

**Comment by xunleii** [comment]
This module has been released to the TF registry (v3.1.0)
